### PR TITLE
fix UAT feedback

### PR DESCRIPTION
**Pull request by hakiemaul**

### CODE REVIEW

File: xendit.php [hunk 1]
```diff
@@ -1087,6 +1096,50 @@ private function _loadXenditInterface() {
...
+			$item['price'] = ceil($line_item_price * $conversion_rate);
+			$item['name'] = $individual_item->order_item_name;
+			$item['category'] = $individual_item->category_name;
+			$item['url'] = $individual_item->product_url;
+			$items_details[] = $item;
+		}
+
+		$fname = $order['details']['BT']->first_name;
+		if (isset($order['details']['BT']->middle_name) and $order['details']['BT']->middle_name) {
+			$fname .= $order['details']['BT']->middle_name;
```
Comment (IreneGohtami) [inline xendit.php]: Add a space before appending the middle name?

File: xendit.php [hunk 2]
```diff
@@ -1087,6 +1096,50 @@ private function _loadXenditInterface() {
...
+		}
+		$lname = $order['details']['BT']->last_name;
+		$address = $order['details']['BT']->address_1;
+		if (isset($order['details']['BT']->address_2) and $order['details']['BT']->address_2) {
+			$address .= $order['details']['BT']->address_2;
```
Comment (IreneGohtami) [inline xendit.php]: Add a new line before appending the address_2

File: xendit/tmpl/cc_payment.php [hunk 3]
```diff
@@ -73,78 +79,86 @@
...
-						"is_multiple_use"	: true
-					};
-
-					Xendit.setPublishableKey('<?php echo $viewData['public_key']; ?>');
-
-					$('#year').val(data.card_exp_year);
-					$('#month').val(data.card_exp_month);
-					$('#card_cvn').val(data.card_cvn);
+			if (useClick){
+				useClick = false;
```
Comment (hakiemaul) [inline xendit/tmpl/cc_payment.php]: @candrasaputra @IreneGohtami i add useClick here to reset the behavior. wdyt?

Comment (candrasaputra) [reply xendit/tmpl/cc_payment.php]: oh yeah, it's important one, i think it must.

**Pull request merged**